.env.example

```diff
@@ -1,3 +1,8 @@
 VITE_CLOUDINARY_API_KEY=
 VITE_CLOUDINARY_SECRET=
-VITE_BIBLE_API_KEY=
\ No newline at end of file
+VITE_BIBLE_API_KEY=
+VITE_DO_ACCESS_KEY_ID=
```

Comment:
To keep the service provider generic, we should rename these to `VITE_S3_ACCESS_KEY_ID` etc.